src/components/Footer/institutions.ts

```diff
@@ -1,12 +1,12 @@
-import logoTramontinaNegativo from '@assets/logoTramontinaNegativo.png';
-import logoSabanciNormal from '@assets/logoSabanciNormal.png';
-import logoUfrgsNegative from '@assets/logoUfrgsNegativo.png';
-import logoUfgNegative from '@assets/logoUfgNegativo.png';
-import logoFgvNegative from '@assets/logoNegativaFGV.png';
-import logoMitNegative from '@assets/logoNegativaMIT.png';
-import logoBologna from '@assets/logoBologna.png';
-import logoUnb from '@assets/logoUnb.png';
-import logoNCF from '@assets/logoNCF.png';
+import logoTramontinaNegativo from '@assets/institutions/logoTramontinaNegativo.png';
```

Comment:
**nitpick:** with the new `images` folder, maybe can be interesting create a relative path direct to that folder